### PR TITLE
May 5 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | e585513 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-294.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20230505_amplify.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+#### Feature updates for Product Foundry
+
+* **Product visibility restricted to authenticated users** - The product visibility has been enhanced to allow provider forbid access to product when the user are not authenticated in public Marketplace.
+* **New currency available for paid plan: Brazilian real** - The Brazilian real has been added to the list of available currencies for paid plan.
+* **Create a new product version from an archived product** - Catalog Manager are now allowed to create a new product version from an archived product.
+
+#### Bug fixes for Product Foundry
+
+* **Marketplace name synchronization** - Before the fix, the Marketplace title from marketplace provider object in API Server model were not synchronized with the title entered in the Marketplace settings. Now every change made on the marketplace settings title is reflected in the marketplace object on the provider side (refer to `axway central get marketplaces`).
+* **Plan details screen** - The plan details screen displays correctly the quota type (Standard, Tiered-volume, Tiered=graduated and Pa Per Use) as well as the currency associated with it.
```
Comment (robertengaxway) [inline content/en/docs/amplify_relnotes/20230505_amplify.md:111]: typo "Pay Per Use"

File: content/en/docs/amplify_relnotes/20230505_amplify.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+There are no new features, bug fixes, or enhancements in this release.
+
+### Product Foundry
+
+The following enhancements and bug fixes are available in this release.
+
+#### Feature updates for Product Foundry
+
+* **Product visibility restricted to authenticated users** - The product visibility has been enhanced to allow provider forbid access to product when the user are not authenticated in public Marketplace.
+* **New currency available for paid plan: Brazilian real** - The Brazilian real has been added to the list of available currencies for paid plan.
```
Comment (robertengaxway) [inline content/en/docs/amplify_relnotes/20230505_amplify.md:105]: make currency upper case on the R
Brazilian Real

File: content/en/docs/amplify_relnotes/20230505_amplify.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+* **Navigation menu behavior** - New option are available for the navigation/help menus: they can be visible either when user is logged in or when user is logged out or both. They can also be opened in a separated tab or no.
+* **Rotating credential** - a new option is available for active credentials to enable to create a new credentials based on an existing one. Like this when a credential is about to expire, you can easily replace it with a new one that will have the same characteristics.
+* **New currency available for paid plan: Brazilian Real** - The Brazilian Real has been added to the list of available currencies for paid plan.
+* **Marketplace subscription and team visibility** - When a user is part of multiple teams and subscribe to a product visible to only a portion of his team, then the subscription screen allows the subscriber to select only one of the team that can access the product.
+* **Can not subscribe on Firefox** - the recent version of Firefox strips the authentication header which prevent the marketplace proxying the platform API to retrieve the team list. Now, platform APIs are call directly without any proxy to retrieve the team list.
```
Comment (robertengaxway) [inline content/en/docs/amplify_relnotes/20230505_amplify.md:141]: should we indicate version number of browser if we know it.   
version x.x.x or later

File: content/en/docs/amplify_relnotes/20230505_amplify.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+#### Bug fixes for Marketplace
+
+* **Bug** - Description.
```
Comment (robertengaxway) [inline content/en/docs/amplify_relnotes/20230505_amplify.md:145]: update this to say "There are no new bug fixes for the Marketplace"

Comment (cbordier) [reply content/en/docs/amplify_relnotes/20230505_amplify.md]: Oups, I misplaced few defects.

**Review by robertengaxway — APPROVED** [review]
approved

**Review by cbordier — APPROVED** [review]
I approve the changes

**Pull request merged**